### ISSUE TITLE
[1.18 / SUGGESTION] Layered spheres stone generation.

### ISSUE THREAD

**Issue by KKotik** [PersonTheCat/OreStoneVariants#153]
**Suggestion for 1.18 version of the mod to have new option to create layered spheres stone generation.**
In recent create updates there is a new feature of stone generations, similar to spheres but with repeating pattern stacked horizontal or vertical. It is a good feature on itself, but lacking in control a lot. With it turned on, there is deepslate can generate in default stone layers almost in surface. Spam generation that not related on surface or cave biomes.

As for OSV I suggest that feature to have that config options:
* *Block that can be replaced: [list]
Maximum R: <value> (Maximum radius for both X and Z, but still picked randomly for both independently)
Minimum R: <value> (Maximum radius for both X and Z, but still picked randomly for both independently)
Maximum Y: <value>
Minimum Y: <value>
Repeating pattern: [list]
Type: [horizontal, vertical]
Biome whitelist: [list]* *

**Comment by PersonTheCat** [comment]
This is an excellent idea. We've discussed this already, but for anyone reading: this feature will be coming to a future version of Cave Generator (which is being renamed in 1.18) _before_ OSV. That will be the ideal place to find giant underground features like this. After that, I'll look into porting it later on.

**Comment by KKotik** [comment]
Make sence, should I reopen that one on CG github then?

**Comment by PersonTheCat** [comment]
No, this is good for now. Thanks! The feature is kind of possible in CG already, but I've got a much better way to do it for CG 2.0. CG 2.0 will be so much different from 1.0 that I'll most likely create a new repository and mod page on CurseForge for it.
